### DISCUSSION TITLE
How to configure a syslog drain?

### DISCUSSION THREAD

**Discussion by ckrailo** [vercel/community#3027] [Feature Question] [answered]
### Question

The docs at https://vercel.com/docs/concepts/observability/log-drains-overview/log-drains-reference#syslog-drain suggest I can select a syslog drain type, but I only see json/ndjson options. What am I missing?

![image](https://github.com/vercel/community/assets/51341/613d5dab-996f-48c3-8693-150cfd1e0338)

**Comment by amyegan** [comment, marked as answer]
Hi @ckrailo. I checked with the log drains team and they shared that syslog is only available to integrations. It is not an option for configurable log drains. 

Thank you for bringing this to my attention. I opened a ticket to make this documentation clear regarding what options are available. Sorry for the inconvenience. 🙇
